extensions/amp-ad-network-adzerk-impl/0.1/amp-ad-network-adzerk-impl.js

```diff
@@ -163,6 +163,10 @@ export class AmpAdNetworkAdzerkImpl extends AmpA4A {
         this.creativeMetadata_['customElementExtensions'].push('amp-analytics');
       }
     }
+    if (this.creativeMetadata_['customElementExtensions']
+        .indexOf('amp-mustache') < 0) {
+      this.creativeMetadata_['customElementExtensions'].push('amp-mustache');
```

Comment:
It's a little bit concern to me because we need to make sure to add the `amp-mustache` script before we start render template. I can tell that the order here is 1. add amp-mustache to metadata 2. create friendly iframe with metadata and insert amp-mustache script 3. start render template. 
However it would be pretty hard to enforce the order in future 😢

Reply:
I think we should just remove the checkTemplateDeclared_ call in template-impl.js